lua/telescope/pickers.lua

```diff
@@ -516,6 +516,24 @@ function Picker:hide_preview()
   -- 2. Resize prompt & results windows accordingly
 end
 
+function Picker:reset_prompt()
```

Comment:
this function has a duplicate. https://github.com/nvim-telescope/telescope.nvim/blob/bfd384fe39e6c79c9530cffcaec4d043f488790c/lua/telescope/pickers.lua#L664

Reply:
Good catch. Thanks :)

This one is obsolete. I basically already put it in master because sunjon needed it for arecibo and forgot about it when rebasing :laughing:

Reply:
Did you try it @elianiva? Does everything work as expected?

Reply:
yes, everything works as expected. GJ! :)